airbyte-integrations/connectors/source-mysql/metadata.yaml

```diff
@@ -9,7 +9,7 @@ data:
   connectorSubtype: database
   connectorType: source
   definitionId: 435bb9a5-7887-4809-aa58-28c27df0d7ad
-  dockerImageTag: 3.10.0-rc.4
```

Comment:
i wondered why did we use rc tag in the previous version.. we could just use the minor version ...

Reply:
@Marius was running a progressive rollout right before the break for a datatype conversion fixes ([49918](https://github.com/airbytehq/airbyte/pull/49918)).
I'll make sure this isn't conflicting with the rollout before merging my change.